e2e/test/scenarios/admin/datamodel/hide_tables.cy.spec.js

```diff
@@ -16,15 +16,15 @@ describe("scenarios > admin > datamodel > hidden tables (metabase#9759)", () =>
 
   it("hidden table should not show up in various places in UI", () => {
     // Visit the main page, we shouldn't be able to see the table
-    cy.visit(`/browse/${PRODUCTS_ID}`);
+    cy.visit(`/browse/${SAMPLE_DB_ID}`);
```

Comment:
this was a mistake and it accidentally works because `producs_id` and `sample_db_id` had the same id (=1)

Reply:
🤦‍♂️